src/dotnet/commands/dotnet-nuget/NuGetCommandParser.cs

```diff
@@ -5,74 +5,52 @@
 
 namespace Microsoft.DotNet.Cli
 {
+    // This parser is used for completion and telemetry.
+    // See https://github.com/NuGet/NuGet.Client/tree/dev/src/NuGet.Core/NuGet.CommandLine.XPlat for the actual implementation.
```

Comment:
this link will get out of date very quick. This points to their master and a simply folder shuffle will kill it. i think just a description is enough.